src/main/java/core/basesyntax/services/impl/DataProcessingImpl.java

```diff
@@ -0,0 +1,66 @@
+package core.basesyntax.services.impl;
+
+import core.basesyntax.Operation;
+import core.basesyntax.operationhandlers.OperationHandler;
+import core.basesyntax.services.DataProcessing;
+import core.basesyntax.storage.Storage;
+import core.basesyntax.strategy.OperationStrategy;
+import core.basesyntax.strategy.OperationStrategyImpl;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class DataProcessingImpl implements DataProcessing {
+    private static final int OPERATION_TYPE = 0;
+    private static final int FRUIT_TYPE = 1;
+    private static final int AMOUNT = 2;
+    private static final String COMMA = ",";
+    private final OperationStrategy fruitStrategy;
+    private Storage storage;
+
+    public DataProcessingImpl(OperationStrategyImpl fruitStrategy, Storage storage) {
+        this.fruitStrategy = fruitStrategy;
+        this.storage = storage;
+    }
+
+    @Override
+    public List<String> processData(List<String> enterList) {

```

Comment:
better create a separate class representing transaction objects and return `List<FruitTransaction>`  in this method 
```
public class FruitTransaction {
    private final Operation operation;
    private final String fruit;
    private final int quantity;
 ...
}
```
After in the separate service iterate by `List<FruitTransaction>` and process each transaction by operation